src/mscorlib/src/System/ArgIterator.cs

```diff
@@ -133,5 +133,54 @@ public override bool Equals(Object o)
             
         private IntPtr ArgPtr;                  // Pointer to remaining args.
         private int    RemainingArgs;           // # of remaining args.
+#else
+        public ArgIterator(RuntimeArgumentHandle arglist)
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator see: https://github.com/dotnet/coreclr/issues/9204.
```

Comment:
Nit: it would be nice to add the comment above the `throw` to avoid a super long line.